src/main/java/core/basesyntax/FruitStorageStrategy.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import core.basesyntax.operation.Operation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitStorageStrategy {
```

Comment:
this class should be responsible only for storing Operations, so it would be better to add here your map initialization with operations and remove method "storageOperation()" from class